businessRules/ideas.js

```diff
@@ -42,9 +42,15 @@ function ideasStatusApproval(ideaId, approvalStatus) {
     const ideas = getAllIdeas();
     const idea = ideas.find((idea) => idea.id === ideaId);
 
+    if (!idea) {
+        throw new Error(`Error. No idea with such ID found.`);
+    }
+
     if (approvalStatus === 'approved') {
         updateIdea(ideaId, 'approve', 'approved');
-    } else {
+    }
+
+    if (approvalStatus === 'Not approved') {
```

Comment:
What are all the possible values for `approvalStatus`?

Reply:
Sorry, I thought, that I've sync--ed it. Here how it should look, as I think it should. 
status should have 2 values-  "approved" or "not approved". 
![Screenshot_20](https://github.com/VladimirKarin/Fund-Raising-Project-Server-Side/assets/105012251/696a068a-eca2-4fab-91ce-cb7f571c5f6b)

Reply:
If `approvalStatus` can have only two values, then why not make it a boolean?

You can rename `approvalStatus` to `isApproved` and assign `true` or `false` to it.

Reply:
Updated and tested. And thanks, never thought about that in that SIMPLE way. My bad.